README.md

```diff
@@ -1,2 +1,21 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+```Password = The chosen password that the user must find
+UserChosenWord = the word the user clicked/chose
+SameLetters = 0
+numbLetrs = the number of letters in Password
+
+If UserChosenWord =! Password
```

Comment:
I assume `=!` means "is not equal to", however this is not a standard notation. Did you mean `!=`?